source/codegen/metadata/nidaqmx/__init__.py

```diff
@@ -1,7 +1,7 @@
-from functions import functions
-from attributes import attributes
-from enums import enums
-from config import config
+from .functions import functions
```

Comment:
Note: the original implementation of this is sort of a hack. References within a package either need to include the package name (generally recommended) or be explicitly relative (like this, because it allows all of these init.pys to be identical). The previous implementation "worked" by munging sys.path and *doesn't work* if you try to do it twice in one process (because the first global "functions" wins).